internal/clioptions/clioptions.go

```diff
@@ -213,6 +215,12 @@ func (o *CLIOptions) AddMarketplaceItemIDFlag(flags *pflag.FlagSet) (flagName st
 	return
 }
 
+func (o *CLIOptions) AddPublicFlag(flags *pflag.FlagSet) (flagName string) {
+	flagName = "public"
+	flags.BoolVarP(&o.Public, flagName, "p", false, "specify if the command should fetch also public items")
```

Comment:
If public is alternative to the company-id one, why here we say "fetch **also**"?

Reply:
fixed the mismatch between flag description and documentation